### DISCUSSION TITLE
Unterschied zwischen "generic" und "generic_eps32"? Was beachten?

### DISCUSSION THREAD

**Discussion by NintendoTito** [tbnobody/OpenDTU#1619] [Q&A] [answered]
Ich nutze bzw. administriere zwei OpenDTU. Die erste habe ich im April mit der „alten“ Datei „generic“ (ohne Zusatz) geflasht, die zweite irgendwann im September mit der neuen „generic_esp32“ Datei. Beide DTUs funktionieren, die „alte“ Version steht bei meinen Eltern.

Nach dem Prinzip „Never Change a running System“ habe ich die alte Version schon lange nicht mehr aktualisiert. Wenn ich das nun mal machen möchte, worauf muss ich achten ohne etwas zu bricken? **Wo ist eigentlich der genaue Unterschied zwischen den Dateien mit und ohne Zusatz?**

**Sehe ich dass richtig, dass es zwingend notwendig ist für ein Update die passende Datei zu nutzen, also die alte Version mit der Datei ohne Zusatz und bei der anderen die Datei mit „_esp32“ zu nutzen?**

**Comment by teletapi** [comment]
Gerade hat mir noby den Link geschickt weils mir genauso ging und ich den Unterschied nicht wusste

https://tbnobody.github.io/OpenDTU-docs/firmware/flash_esp/#overview-of-pre-compiled-binaries

**Reply by NintendoTito** [reply]
Danke für den Link. Kenne ich grundsätzlich auch schon. Und man sieht, dass die Datei ohne den Zusatz veraltet ist. Aber was mache ich denn dann mal, wenn die Version nicht mehr weiter fortgeführt wird? Kann ich ne OpenDTU die mit der alten Version geflasht wurde dann auch mit einer anderen Version aktualisieren? Oder muss ich dann alles neu machen?

**Reply by tbnobody** [reply, marked as answer]
Ja kannst du. Anschließend spielst du [die](https://raw.githubusercontent.com/tbnobody/OpenDTU/master/docs/DeviceProfiles/nodemcu_esp32.json) Datei ein (Device Profile) und wählst als Profil das NRF24 aus.
